mmseg/models/losses/cross_entropy_loss.py

```diff
@@ -193,13 +252,15 @@ def forward(self,
             class_weight = cls_score.new_tensor(self.class_weight)
         else:
             class_weight = None
+        # Note: In cls_criterion, label < 0 is invalid.
         loss_cls = self.loss_weight * self.cls_criterion(
             cls_score,
             label,
             weight,
             class_weight=class_weight,
             reduction=reduction,
             avg_factor=avg_factor,
+            avg_non_ignore=self.avg_non_ignore,
             **kwargs)
```

Comment:
The argument `ignore_index` should be explicit instead of using **kwargs.

Reply:
Well, actually it is modified to `**kwargs` 7 days ago. https://github.com/open-mmlab/mmsegmentation/pull/1388#discussion_r828922720.

Reply:
> Well, actually it is modified to `**kwargs` 7 days ago. [#1388 (comment)](https://github.com/open-mmlab/mmsegmentation/pull/1388#discussion_r828922720).

For what reason?

Reply:
Maybe it is unnecessary to be explicit? I agree with your comment, let me roll back first.

Reply:
> 

I just follow the previous code